From 7785c4a0c8d5300edd1fc4a7819e22d24f6491a8 Mon Sep 17 00:00:00 2001
From: Jukka Kaartinen <jukka.o.kaartinen@intel.com>
Date: Mon, 27 May 2013 10:00:55 +0300
Subject: [PORT FROM MAIN] atomisp: Add support for V4L2_EVENT_ATOMISP_3A_STATS_READY -event

BZ: 87056

Add an event to inform user-space of availability of 3A statistics.

This addition allows to decouple timing of preview frames, from 3A
statistics availability. Ideally user-space could start running 3A
earlier by subscribing to this event.

Change-Id: I38b327470d936096bc71c947954e6d9853cd3a3d
Orig-Change-Id: I26fff8df2d2a6f0825aaf1e0c6ec18589a54c300
Signed-off-by: Jukka Kaartinen <jukka.o.kaartinen@intel.com>
Reviewed-on: http://android.intel.com:8080/109689
Reviewed-by: Patale, Pavan <pavan.patale@intel.com>
Tested-by: Patale, Pavan <pavan.patale@intel.com>
Reviewed-by: buildbot <buildbot@intel.com>
Tested-by: buildbot <buildbot@intel.com>
---
 libc/kernel/common/linux/atomisp.h |    3 ++-
 1 files changed, 2 insertions(+), 1 deletions(-)

diff --git a/libc/kernel/common/linux/atomisp.h b/libc/kernel/common/linux/atomisp.h
index a57ea3a..4019da2 100644
--- a/libc/kernel/common/linux/atomisp.h
+++ b/libc/kernel/common/linux/atomisp.h
@@ -712,10 +712,11 @@ struct v4l2_private_int_data {
 #define V4L2_BUF_FLAG_BUFFER_VALID 0x0800
 /* WARNING: DO NOT EDIT, AUTO-GENERATED CODE - SEE TOP FOR INSTRUCTIONS */
 #define V4L2_BUF_TYPE_VIDEO_CAPTURE_ION (V4L2_BUF_TYPE_PRIVATE + 1024)
+#define V4L2_EVENT_ATOMISP_3A_STATS_READY (V4L2_EVENT_PRIVATE_START + 1)
 enum {
  V4L2_COLORFX_SKIN_WHITEN_LOW = 1001,
- V4L2_COLORFX_SKIN_WHITEN_HIGH = 1002,
 /* WARNING: DO NOT EDIT, AUTO-GENERATED CODE - SEE TOP FOR INSTRUCTIONS */
+ V4L2_COLORFX_SKIN_WHITEN_HIGH = 1002,
 };
 #endif
 
-- 
1.7.4.1

